lib/index.js

```diff
@@ -440,7 +497,7 @@ exports.init = function (sequelize, optionsArg) {
         underscored: options.underscored
       });
       Revision.associate = function associate(models) {
-        Revision.belongsTo(sequelize.model(options.userModel));
+        Revision.belongsTo(sequelize.model(options.userModel), options.belongsToUserOptions);
```

Comment:
This works even if `options.belongsToUserOptions` is null? Actually it's okay, as this will be in the next major version.

Reply:
Was this not addressed in a different pr already? I could just make it work like that one proposed